python/py-pymysql/Portfile

```diff
@@ -4,22 +4,22 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-pymysql
-version             0.8.0
+version             0.9.2
 license             MIT
 platforms           darwin
 supported_archs     noarch
 maintainers         {@xeron gmail.com:xeron.oskom} openmaintainer
 description         Pure-Python MySQL client library
 long_description    ${description}
 
-python.versions     27 34 35 36
+python.versions     27 34 35 36 37
 
 homepage            https://pypi.python.org/pypi/${python.rootname}/
 master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
 distname            PyMySQL-${version}
 
-checksums           rmd160  6bb48cc4b0028666b94bfedb36f3841bb9f34a10 \
-                    sha256  32da4a66397077d42908e449688f2ec71c2b18892a6cd04f03ab2aa828a70f40
+checksums           rmd160  6c5c1015e61591e59927b4b8588acf274b6e6edd \
+                    sha256  9ec760cbb251c158c19d6c88c17ca00a8632bac713890e465b2be01fdc30713f
```

Comment:
It is now customary to put in a "size" field in addition to the rmd160 and sha256.